.github/workflows/pr-demo.yml

```diff
@@ -42,49 +41,41 @@ jobs:
           key: ${{ runner.os }}-${{ hashFiles('yarn.lock') }}
 
       - name: Install
-        run: |
-          yarn install --frozen-lockfile
-          yarn global add surge
-          echo "$(yarn global bin)" >> $GITHUB_PATH
-
+        working-directory: ./
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_READ_TOKEN }}
+        run: yarn install --frozen-lockfile
 
       - name: Build
-        run: |
-          yarn test:demo
-          ./.github/workflows/before-surge.sh
+        run: yarn test:demo
         env:
           STORYBOOK_FIGMA_ACCESS_TOKEN: ${{ secrets.STORYBOOK_FIGMA_ACCESS_TOKEN }}
 
-      - name: deploy if master
-        if: github.ref == 'refs/heads/master'
-        run: surge --project .static --domain "talend.surge.sh"
-        env:
-          SURGE_LOGIN: ${{ secrets.SURGE_LOGIN }}
-          SURGE_TOKEN: ${{ secrets.SURGE_TOKEN }}
-
-      - name: Deploy to Netlify production
```

Comment:
I think we need to keep and fix netlify

Reply:
yep, but it is the file for PR, and from what I see we only want netlify for master, so it will be in the workflow for the master branch

Reply:
no, this workflow is also called on master branch